exchangecommon/node_management_status.go

```diff
@@ -2,18 +2,50 @@ package exchangecommon
 
 import (
 	"fmt"
+	"math/rand"
+	"time"
 )
 
 type NodeManagementPolicyStatus struct {
-	ScheduledTime   string `json:"scheduledTime"`
-	ActualStartTime string `json:"startTime,omitempty"`
-	CompletionTime  string `json:"endTime,omitempty"`
-	UpgradedVersion string `json:"upgradedVersion"`
-	Status          string `json:"status"`
-	ErrorMessage    string `json:"errorMessage,omitempty"`
+	ScheduledTime     string `json:"scheduledTime"`
+	scheduledUnixTime time.Time
+	ActualStartTime   string `json:"startTime,omitempty"`
+	CompletionTime    string `json:"endTime,omitempty"`
+	UpgradedVersion   string `json:"upgradedVersion"`
+	Status            string `json:"status"`
+	ErrorMessage      string `json:"errorMessage,omitempty"`
```

Comment:
Should add an attribute "AgentUpgrade" under `NodeManagementPolicyStatus` and put all the other attributes under `AgentUpgrade`. See https://github.com/open-horizon/exchange-api/issues/539. This will allow us to add other node management types into a nmp.